clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -91,7 +93,8 @@ def start_clipper(
             compability and preserve the expected behavior of the system.
         cache_size : int, optional
             The size of Clipper's prediction cache in bytes. Default cache size is 32 MiB.
-
+        config_file: str(optional)
+            File for the configuration of the application or model
```

Comment:
Expand on this parameter documentation: 

1. Indicate that `config_file` specifies the path to a configuration file.
 
2. Explain that, if the path is present, Clipper will deploy models, register applications, and create links as specified by the file. 

3. Finally, provide the name of the sample configuration file that you wrote so that the user can reference it.

Reply:
In Progress

Reply:
Done